### ISSUE TITLE
Wrong indent in multil-line var definition with types

### ISSUE THREAD

**Issue by ogdenwebb** [nim-lang/nim-mode#199]
Conforming Bugs
===============

Please tell us below:

- [x] basic information
   - Gentoo Linux, kernel 4.17.4
   - Emacs GNU Emacs 26.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.30) of 2018-08-24
   - Nim and nimsuggest version 0.18.0
- [x] Are you using latest version of nim-mode? Yes, 0.4.2 from MELPA.
- [ ] Are there message related to nim-mode in `*Messages*` buffer? No.
- [x] Did you read README.md?
- [ ] Is your problem related to nimsuggest-mode? (company-mode, flycheck, or el-doc)
   - [ ] Did you try nim-mode without nimsugggest-mode? What happened?
         (Comment out nimsuggest-mode's config please)

**Expected behavior:**

```nim
var
  x, y: string
  a, b: float
```

**Actual behavior:**

```nim
# after calling 'electric-and-indent' or 'indent-region'
var
  x, y: string
a, b: float
```

But with a value it works well:

```nim
var
  x, y: string = "test string"
  a, b: float
```

**Comment by krux02** [comment]
put `(electric-indent-mode 0)` from your `nim-mode-hook`.  Electric indent has more severe problems thas this.  Sometimes it blocks emacs for several seconds and then automatically screws up the entire file.  It's not worth the trouble.

**Comment by ogdenwebb** [comment]
@krux02 it doesn't change anything for me. With enabled and disabled electric-indent-mode in nim-mode hook the issue appears here.

As a sidenote, nim-indent-calculate-indentation returns 0 on the third line of code.

**Comment by krux02** [comment]
@ogdenwebb you are totally right, I just tested it.  My recommendation is to use ``var`` on every line.  This doesn't fix but it makes the bug non relevant.

```nim
var  x, y: string
var a, b: float
```
